src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -559,19 +749,26 @@ void MenuGame::initInput()
 
     game.addInput(
         Config::getTriggerRotateCCW(),
-        [this](ssvu::FT /*unused*/) { leftAction(); }, t::Once);
+        [this](ssvu::FT /*unused*/) { leftAction(); }, t::Once, Tid::RotateCCW);
     game.addInput(
         Config::getTriggerRotateCW(),
-        [this](ssvu::FT /*unused*/) { rightAction(); }, t::Once);
+        [this](ssvu::FT /*unused*/) { rightAction(); }, t::Once,
+        Tid::RotateCW);
     game.addInput(
-        Config::getTriggerUp(), [this](ssvu::FT /*unused*/) { upAction(); },
+        {{k::Up}}, [this](ssvu::FT /*unused*/) { upAction(); }, //hardcoded
```

Comment:
Ditto, why should this be hardcoded?

Reply:
The idea is that no matter how much the user fools around with the binds, it should never be able to "screw itself over".
For example, if we were to not have hardcoded Up and Down binds, and user were to unassign all keys from those action, it would practically be unable to navigate the menues anymore.
Therefore, unless this explanation has not been convincing, I strongly feel we should at least keep the most essential hardcoded inputs I created.